game-core/src/main/java/games/strategy/engine/framework/startup/ui/pbem/PbemSetupPanel.java

```diff
@@ -44,6 +47,7 @@
   private final GameSelectorModel gameSelectorModel;
   private final DiceServerEditor diceServerEditor;
   private final ForumPosterEditor forumPosterEditor;
+  private final ForumPosterEditorViewModel forumPosterEditorViewModel;
```

Comment:
This class is just updated to insantiate a view model, pass that to the PBF view, and then instead of interacting with the PBF view for 'are all fields valid', we query the view model instead.